apps/site/src/app/admin/dashboard/AdminDashboard.tsx

```diff
@@ -2,11 +2,17 @@
 
 import Container from "@cloudscape-design/components/container";
 import ContentLayout from "@cloudscape-design/components/content-layout";
+import SpaceBetween from "@cloudscape-design/components/space-between";
+
+import ApplicantSummary from "./components/ApplicantSummary";
 
 function AdminDashboard() {
 	return (
 		<ContentLayout>
-			<Container>Admin Dashboard</Container>
+			<SpaceBetween size="l">
+				<Container>Admin Dashboard</Container>
+				<ApplicantSummary />
```

Comment:
Issue: For non-admin roles, such as `organizer` and `volunteer`, they'll see a red x in the center of the screen, which at least to me, makes it seem like the chart failed to load. Would it be better to just hide it for non-admin roles?

<img width="958" alt="image" src="https://github.com/HackAtUCI/irvinehacks-site-2024/assets/28026520/28450f3d-76a2-4c50-84da-7c5bc7cad80b">

Reply:
Good catch, forgot to consider the new roles.